src/index.ts

```diff
@@ -98,8 +99,8 @@ const languageFilenameMap: any = {
 
 export default async (generatorOptions: IGeneratorOptions) => {
   const { openrpcDocument, outDir } = generatorOptions;
-
-  const methodTypings = new MethodTypings(openrpcDocument);
+  const dereffed = await parseOpenRPCDocument(openrpcDocument);
```

Comment:
wouldnt this change make it deref twice on `cli` usage?

Reply:
fixed